Description: Support for olf plexus-utils 1.5.x (as currently in Debian).
Author: Damien Raude-Morvan <drazzib@debian.org>
Forwarded: no
Last-Update: 2011-10-23

--- a/src/main/java/org/apache/felix/bundleplugin/BundlePlugin.java
+++ b/src/main/java/org/apache/felix/bundleplugin/BundlePlugin.java
@@ -734,7 +734,7 @@
             {
                 getLog().info( "Writing BND instructions to " + dumpInstructions );
                 dumpInstructions.getParentFile().mkdirs();
-                FileUtils.fileWrite( dumpInstructions, "# BND instructions" + NL + buf );
+                FileUtils.fileWrite( dumpInstructions.getAbsolutePath(), "# BND instructions" + NL + buf );
             }
         }
 
@@ -748,7 +748,7 @@
             {
                 getLog().info( "Writing BND classpath to " + dumpClasspath );
                 dumpClasspath.getParentFile().mkdirs();
-                FileUtils.fileWrite( dumpClasspath, "# BND classpath" + NL + buf );
+                FileUtils.fileWrite( dumpClasspath.getAbsolutePath(), "# BND classpath" + NL + buf );
             }
         }
     }
